src/App.tsx

```diff
@@ -38,6 +44,32 @@ const App = () => {
     }
   }, [messageManager])
 
+  // https://reactnative.dev/docs/linking?syntax=android#handling-deep-links
+  useEffect(() => {
+    if (!settings) return
+
+    // shared link handler
+    const deepLinkHandler = ({ url }: { url: string }) => {
+      // "shouldn't" happen, but did in testing
+      if (!url) return
+      // e.g. shapeshift://yat/🦊🚀🌈
+      // url escaped http://192.168.1.22:3000/#/yat/%F0%9F%A6%8A%F0%9F%9A%80%F0%9F%8C%88
+      // to test this, run:
+      // npx uri-scheme open "shapeshift://yat/%F0%9F%A6%8A%F0%9F%9A%80%F0%9F%8C%88" --ios
+      // npx uri-scheme open "shapeshift://yat/%F0%9F%A6%8A%F0%9F%9A%80%F0%9F%8C%88" --android
+      const URL_DELIMITER = 'shapeshift://'
+      const path = url.split(URL_DELIMITER)[1]
+      const newUri = `${settings?.SHAPESHIFT_URI}/#/${path}?${Date.now()}`
```

Comment:
`?Date.now()` tricks the webview into navigating to a different url. without it, the urls are the same, even if the webview has routed to some other page within the webview